.github/workflows/publish.yml

```diff
@@ -44,14 +44,15 @@ jobs:
           restore-keys: |
             ${{runner.os}}-yarn-
 
+      - run: corepack yarn install --immutable
+
+      - run: corepack yarn pack --out corepack.tgz
+
       - name: Publish to the npm registry
-        run: |
-          corepack yarn install --immutable
-          corepack yarn npm publish
+        run: npm publish corepack.tgz --tag latest --provenance
```

Comment:
What's the reason to specify `latest` here?

Reply:
You're right - it's latest by default, so it's useless. Removed.